test/javascripts/acceptance/topic-edit-timer-test.js.es6

```diff
@@ -253,3 +259,52 @@ QUnit.test("auto delete", assert => {
     assert.ok(regex.test(html));
   });
 });
+
+QUnit.test(
+  "Manually closing before the timer will clear the status text",
+  async assert => {
+    // prettier-ignore
+    server.post("/t/280/timer", () => // eslint-disable-line no-undef
+      response({
+        success: "OK",
+        execute_at: new Date(
+          new Date().getTime() + 1 * 60 * 60 * 1000
+        ).toISOString(),
+        duration: 1,
+        based_on_last_post: false,
+        closed: false,
+        category_id: null
+      })
+    );
+
+    // prettier-ignore
+    server.put("/t/internationalization-localization/280/status", () => // eslint-disable-line no-undef
```

Comment:
When eslint and prettier do not agree with each other LOL. 

Should be ✅ this time 🤞

Reply:
what about `() => {}` here instead then?